examples/dynamic-config-cp/Dockerfile-control-plane

```diff
@@ -1,6 +1,10 @@
 FROM golang
 
-RUN apt-get -y update && apt-get install -y -qq --no-install-recommends netcat
+RUN apt-get update \
```

Comment:
afaict netcat isnt requred at all - not even in the host

Reply:
seems i was wrong about this - its used in the docker-compose file for healthcheck - apologies